plugins/google_analytics/girder_google_analytics/web_client/views/ConfigView.js

```diff
@@ -56,6 +56,8 @@ var ConfigView = View.extend({
             },
             error: null
         }).done(() => {
+            location.reload();
```

Comment:
What's this for?

Reply:
This clears the previous js bundle from google. If we change the tag we're tracking, we need to request a fresh js bundle from `main.js:initGoogleAnalytics`

Reply:
@zachmullen is this an issue?

Reply:
I'd say it's generally an anti-pattern to do a hard refresh in a single-page app. Is it not sufficient to re-invoke `initGoogleAnalytics`? Or would the presence of the previous state corrupt the page in some way?

Reply:
I was initially worried about the prospect of having multiple competing results from `initGoogleAnalytics`'s `$.getScript`, but after testing it further seems like it won't be an issue for usage.